core/src/test/scala/org/apache/spark/storage/MemoryStoreSuite.scala

```diff
@@ -60,7 +60,7 @@ class MemoryStoreSuite
     SizeEstimator invokePrivate initialize()
   }
 
-  def makeMemoryStore(maxMem: Long): (MemoryStore, BlockInfoManager) = {
+  def makeMemoryStore(maxMem: Long, conf: SparkConf = conf): (MemoryStore, BlockInfoManager) = {
```

Comment:
I think it is for passing a SparkConf with modified `spark.storage.unrollMemoryThreshold`.